### PR TITLE
Add simple merge command #90

### PR DESCRIPTION
**Pull request by m-mohr**
Implements #90

### CODE REVIEW

File: tests/test_merge.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    result = runner.invoke(merge, args)
+    assert result.exit_code == 0, result.output
+
+    # Merged parquet file should be valid
+    result = runner.invoke(validate, [tmp_file.name, '--data'])
+    assert result.exit_code == 0, result.output
+
+    parquet = import_optional_dependency(
+        "pyarrow.parquet", extra="pyarrow is required for Parquet support."
+    )
```
Comment (m-mohr) [inline tests/test_merge.py:28]: Is that an actual issue? We use it in other places, too, so it's not really optional, isn't it?

Comment (ivorbosloper) [reply tests/test_merge.py:28]: I searched in the code on how to open/read/write parquet files (for testing this case). This is what I found in `geopandas.py` and I bluntly assumed this is how I should import it... Feel free to change

**Review by ivorbosloper — APPROVED** [review]
Tested, it works

**Comment by m-mohr** [comment]
Is there a better default?

**Pull request merged**